lib/Compiler.js

```diff
@@ -55,16 +59,20 @@ function compileSolc(embark, contractFiles, cb) {
 
         for (let contractFile in json.contracts) {
           let className = contractFile.substr( contractFile.indexOf(":") + 1);
-          let fileName = contractFile.substr(0, contractFile.indexOf(":"));
+          let filename = contractFile.substr(0, contractFile.indexOf(":"));
           
           let contract = json.contracts[contractFile];
-          
+          if(compiled_object[className] && compiled_object[className].filename != filename){
+            logger.warn(`Duplicated contract '${className}' found. Using '${compiled_object[className].filename}' instead of '${file.filename}'`);
```

Comment:
You want to use "Duplicate" instead of "Duplicated".

Also, what's the side effect here? The new file won't be looked up? I think the message needs to be clearer.